Makefile

```diff
@@ -19,31 +19,42 @@ endif
 help: ## Print the help documentation
 	@grep -E '^[a-zA-Z\/_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: dev
-dev: ## Start development environment
+#
+# ----- END PREAMBLE -----
+#
+
+#
+# ----- START DEVELOPMENT TARGETS -----
+#
+
+.PHONY: dev_up
+dev_up: ## Start development environment
 	docker-compose -f docker-compose.dev.yml build --pull
 	aws-vault exec "${AWS_PROFILE}" -- docker-compose -f docker-compose.dev.yml up -d
 
-.PHONY: dev_exec
-dev_exec: ## Attach to the development environment if running
+.PHONY: dev
+dev: ## Attach to the development environment if running
 	aws-vault exec "${AWS_PROFILE}" -- docker-compose -f docker-compose.dev.yml exec dev /bin/bash
```

Comment:
Got a question on how this will work for devs. When running `make dev` aws-vault will generate a temporary token valid for 1hr passed to the container environments. After that token expires will devs need to do a `down` then `up` to refresh?

Reply:
After an hour you just `Ctrl-C` and then `make dev` again and you should get prompted for new AWS creds. No need to take down the whole environment unless the server itself is trying to talk to AWS, which isn't something this application does.

Reply:
Ah so docker-compose will detect new AWS creds and just rebuild that one container? Nice :)

Reply:
yep. And I just added a note about it to the docs.